fiware-rss/src/main/java/es/upm/fiware/rss/model/RSSReport.java

```diff
@@ -33,6 +34,17 @@
     private String currency;
     private List<StakeholderModel> stakeholders;
 
+
+    private boolean paid;
+
+	public int getId() {
+		return id;
+	}
+
+	public void setId(int id) {
+		this.id = id;
+	}
+
```

Comment:
Review the indentation of this file, it seem that you have included tabs instead of spaces.